test/tokens/mocks/ERC1155InvalidReceiverMock.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+/**
+ * @title ERC1155InvalidReceiverMock
+ * @author jtriley.eth
+ * @notice Implements invalid responses from an `ERC1155Receiver`.
+ * @dev param names are ommitted to avoid unused variable warnings.
```

Comment:
```suggestion
 * @dev param names are omitted to avoid unused variable warnings.
```

Reply:
omitting this since tests will be removed from this pr, though i will update this in the subsequent "tests" pr.